spectree/spec.py

```diff
@@ -163,7 +165,7 @@ def _generate_spec(self):
 
                 routes[path][method.lower()] = {
                     'summary': summary or f'{name} <{method}>',
-                    'operationID': f'{name}__{method.lower()}',
+                    'operationId': camelize(f'{name}', False),
```

Comment:
Does this not need to be unique?  If you don't have the method in this, it'll clash if you've more than one method for the same url.

Reply:
Ah I see what happened - the library simply parses the name of the function that is used to get the name of the OperationId, which in theory you wouldn't be able to collide due to namespacing but I'll put the method back in.